gdx/src/com/badlogic/gdx/utils/JsonMatcher.java

```diff
@@ -16,6 +16,10 @@
 
 package com.badlogic.gdx.utils;
 
+import java.io.InputStream;
```

Comment:
I see InputStream is used somewhere here; would writing to an OutputStream be a reasonable feature to add? It would make testing libGDX Json implementations against others a little easier (as in, we could PR that JVM Json benchmark to include libGDX 1.13.6). If OutputStream support is a hard feature to add, I wouldn't worry about it, though. It might be nice for some usages libGDX doesn't have internally, but users may benefit from.